addons/main/XEH_PREP.hpp

```diff
@@ -9,6 +9,7 @@ PREP(handleArmorDamage);
 PREP(initAIUnit);
 PREP(initPlates);
 PREP(moduleHeal);
+PREP(regenBleedout);
```

Comment:
Should be moved to the else of the ace med check below.

Reply:
Oh right, because down timers aren't a thing with ace med. 👍